Packs/CommonScripts/Scripts/ConvertTimezoneFromUTC/ConvertTimezoneFromUTC.py

```diff
@@ -0,0 +1,43 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+import pytz
+from traceback import format_exc
+
+
+def determine_correct_format(time: str, fmt: str) -> datetime:
+    time = datetime.strptime(time, fmt)
+    return time
+
+
+def convert_UTC_Timezone_command(time: datetime, timezone: str, fmt: str) -> CommandResults:
```

Comment:
Lets change the names to something more meaningful. what is time? the time we want to convert? what is timezone? the timezone of the time we want to convert or a new timezone to convert to?